intermediate_tester.py

```diff
@@ -17,5 +17,21 @@ def test_dictionary(self):
         }
         self.assertTrue(b.dictionary_exercise(book_index, 'apples', 'oranges') == [64,70])
 
+    def test_calculate_GPA_CSV(self):
+        print('Test calculate GPA function')
+        gpa = b.calculate_GPA_CSV('data/GPAData/gpadata.csv')
+        self.assertTrue(round(gpa, 2) == 3.52)
+
+    def test_calculate_statistics(self):
+        print('Test calculate statistics function')
+        average, max_gdp, min_gdp, country_with_highest_gdp, country_with_lowest_gdp, variance, standard_deviation = b.calculate_statistics('data/GDPData/GDPData.csv')
+        self.assertTrue(str(average) == "4.452496609e+11")
```

Comment:
That I'm not sure. I tried rounding too, but it's not very effective. Should I just leave out these comparisons?